File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_DSField.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DSField } from './NS_DSField.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DSFieldService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Ma', 'Ten', 'Viet', 'Loai', 'BangQH', 'TenTable'];

    List: NS_DSField[] | undefined;
    ListFilter: NS_DSField[] | undefined;
    FormData!: NS_DSField;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DSField";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DSFieldService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DSFieldService : BaseService<NS_DSField, INS_DSFieldRepository>
    , INS_DSFieldService
    {
    private readonly INS_DSFieldRepository _NS_DSFieldRepository;
    public NS_DSFieldService(INS_DSFieldRepository NS_DSFieldRepository) : base(NS_DSFieldRepository)
    {
    _NS_DSFieldRepository = NS_DSFieldRepository;
    }
    }
    }

